pwiz_tools/Skyline/Properties/Settings.cs

```diff
@@ -2359,8 +2359,9 @@ public override string GetDisplayName(IonMobilityLibrary item)
             IEnumerable<IonMobilityLibrary> existing, object tag)
         {
             var ionMobilityFilteringUserControl = (owner as IonMobilityFilteringUserControl) ??
-                                                  (owner as TransitionSettingsUI)!.IonMobilityControl;
-            var ionMobilityWindowWidthCalculator = ionMobilityFilteringUserControl!.IonMobilityWindowWidthCalculator;
+                                                  ((owner as TransitionSettingsUI)?.IonMobilityControl) ??  // Accessed via Settings>TransitionSettings>IonMobility>Add
+                                                  ((owner as Form)?.Owner as TransitionSettingsUI)?.IonMobilityControl; // Accessed via Settings>TransitionSettings>IonMobility>EditList>Add|EditCurrent
+                var ionMobilityWindowWidthCalculator = ionMobilityFilteringUserControl!.IonMobilityWindowWidthCalculator;

```

Comment:
This line is indented wrong.

Reply:
I always appreciate your eye for detail, thanks.